core/src/main/scala/org/apache/spark/api/r/RRDD.scala

```diff
@@ -127,6 +127,13 @@ private[r] object RRDD {
       sparkConf.setExecutorEnv(name.toString, value.toString)
     }
 
+    if (sparkEnvirMap.containsKey("spark.r.sql.default.derby.dir") &&
```

Comment:
in further testing, I found that `derby.system.home` affects metastore_db too, but I'm changing to set `derby.stream.error.file` instead